src/main/java/com/dougnoel/sentinel/webdrivers/GridWebDriverFactory.java

```diff
@@ -0,0 +1,32 @@
+package com.dougnoel.sentinel.webdrivers;
+
+import java.net.URL;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import com.dougnoel.sentinel.configurations.Configuration;
+import com.dougnoel.sentinel.exceptions.MalformedURLException;
+
+public class GridWebDriverFactory {
+	
+	private GridWebDriverFactory() {
+		
+	}
+	
+	protected static WebDriver createGridDriver(String browser, String gridUrl) {
+    	var capability = new DesiredCapabilities();
+    	capability.setCapability("browserName", browser);
+        var browserVersion = Configuration.toString("browserVersion");
+    	if (browserVersion != null) 
+    	    capability.setCapability("browserVersion", browserVersion);
+    	
+    	URL url;
+        try {
```

Comment:
This try should be on the same level as the URL line above.

Reply:
That's weird, my eclipse shows it's in the same line